README.md

```diff
@@ -5,167 +5,142 @@
 [![Build Status][1]][2]
 
 ## Overview
-Heptio Ark is a utility for managing disaster recovery, specifically for your [Kubernetes][14] cluster resources and persistent volumes. It provides a simple, configurable, and operationally robust way to back up and restore applications and PVs from a series of checkpoints. This allows you to better automate in the following scenarios:
 
-* **Disaster recovery** with reduced TTR (time to respond), in the case of:
-    * Infrastructure loss
-    * Data corruption
-    * Service outages
+Ark gives you tools to back up and restore your Kubernetes cluster resources and persistent volumes. Ark lets you:
 
-* **Cross-cloud-provider migration** for Kubernetes API objects (cross-cloud-provider migration of persistent volume snapshots not yet supported)
+* Manage disaster recovery.
+* Copy cluster resources across cloud providers. NOTE: PersistenVolume objects are not supported.
+* Replicate your production environment for dev and testing enviornments.
 
-* **Dev and testing environment setup (+ CI)**, via replication of prod environment
+Ark consists of:
 
-More concretely, Heptio Ark combines an in-cluster service with a CLI that allows you to record both:
-1. *Configurable subsets of Kubernetes API objects* -- as tarballs stored in object storage
-2. *Disk snapshots of Persistent Volumes* -- via the cloud provider APIs
+* A server that runs on your cluster
+* A client with a CLI
 
-Heptio Ark currently supports the [AWS][15], [GCP][16], and [Azure][17] cloud provider platforms.
+## Getting started
 
-## Quickstart
+The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-This guide gets Ark up and running on your cluster, and goes through an example using the following:
-* **Minio, an S3-compatible storage service** that runs locally on your cluster. This is the storage service where backup files are uploaded. *Note that Ark is intended to run on a cloud provider--we are using Minio here to keep the example convenient and self-contained.*
+NOTE: Ark is intended to run on a cloud provider. For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. PersistentVolume disk snapshots require integration with a cloud provider API. See [Set up Ark with your cloud provider][3].
```

Comment:
I would prefer we remove the statement that Ark is meant to run in the cloud. There are on-premises storage solutions out there, and Ark can work with them if someone writes a plugin (we, for example, have a prototype Rook plugin).